crates/ra_ide/src/inlay_hints.rs

```diff
@@ -1154,32 +1162,35 @@ fn main() {
             struct A { pub b: B }
             struct B { pub c: C }
             struct C(pub bool);
+            struct D;
+
+            impl D {
+                fn foo() -> i32 { 42 }
+            }
 
             fn main() {
                 let x = A { b: B { c: C(true) } }
                     .b
                     .c
                     .0;
+                let x = D
+                    .foo();
```

Comment:
Foo is an assoc function, not a method currently, so it either has to be called as `D::foo()` or to be declared as `fn foo(&self)`.
I presume, the latter would be more appropriate here.

Reply:
Fixed, thanks.